CONTRIBUTING.md

```diff
@@ -0,0 +1,290 @@
+
+[Back to TOC](#table-of-contents)
+
+### Improving the documentation
+
+The documentation hosted at https://kuma.io/docs is open source and built
+with [Netlify](https://www.netlify.com/). You are welcome to propose changes to it
+(correct typos, add examples or clarifications...)!
+
+The repository is also hosted on GitHub at:
+https://github.com/Kong/kuma-website
+
+[Back to TOC](#table-of-contents)
+
+### Submitting a patch
+
+Feel free to contribute fixes or minor features, we love to receive Pull
+Requests! If you are planning to develop a larger feature, come talk to us
+first!
+
+When contributing, please follow the guidelines provided in this document. They
+will cover topics such as the different Git branches we use, the commit message
+format to use or the appropriate code style.
+
+Once you have read them, and you are ready to submit your Pull Request, be sure
+to verify a few things:
+
+- Your work was based on the appropriate branch (`master` vs. `feature/latest`), 
+  and you are opening your Pull Request against the appropriate one
+- Your commit history is clean: changes are atomic and the git message format
+  was respected
+- Rebase your work on top of the base branch (seek help online on how to use
+  `git rebase`; this is important to ensure your commit history is clean and
+   linear)
+- The tests are passing: run `make test`, `make test konvoy-injector`,
+  `make test kuma-dp`, or whichever is appropriate for your change
+- Do not update CHANGELOG.md yourself. Your change will be included there in
+  due time if it is accepted, no worries!
+
+If the above guidelines are respected, your Pull Request will be reviewed by
+a maintainer.
+
+If you are asked to update your patch by a reviewer, please do so! Remember:
+**you are responsible for pushing your patch forward**. If you contributed it,
+you are probably the one in need of it. You must be prepared to apply changes
+to it if necessary.
+
+If your Pull Request was accepted and fixes a bug, adds functionality, or
+makes it significantly easier to use or understand Kuma, congratulations!
+You are now an official contributor to Kuma. Get in touch with us to receive
+your very own [Contributor T-shirt](#contributor-t-shirt)!
+
+Your change will be included in the subsequent release Changelog, and we will
+not forget to include your name if you are an external contributor. :wink:
+
+[Back to TOC](#table-of-contents)
+
+#### Git branches
+
+We work on two branches: `master`, where non-breaking changes land, and 
+`feature/latest`, where important features or breaking changes land in-between
+major releases.
+
+When contributing to Kuma, this distinction is important. Please ensure that
+you are basing your work on top of the appropriate branch, it might save you
+some time down the road!
+
+If you have write access to the GitHub repository, please follow the following
+naming scheme when pushing your branch(es):
+
+- `feat/foo-bar` for new features
```

Comment:
Why this is `feat/` but branch for breaking changes is `feature/`. Is this intentional?

Reply:
feature branch in docs removed, no longer relevant